src/test/java/CheckCurrencyIsEnabledInShopTest.java

```diff
@@ -0,0 +1,26 @@
+import com.opencart.enums.URLs;
+import com.opencart.navigation.Navigation;
+import com.opencart.steps.*;
+import org.testng.annotations.Test;
+
+public class CheckCurrencyIsEnabledInShopTest extends BaseTest {
+
+    @Test
+    public void editCurrencyValue() {
+        new Navigation().navigateToUrl(URLs.ADMIN_URL.getUrl());
+        LoginAdminPageBL loginAdminPageBL = new LoginAdminPageBL();
+        MainAdminPageBL mainAdminPageBL = loginAdminPageBL.loginToAdminAccount();
+        AdminNavigationBL adminNavigationBL = mainAdminPageBL.getAdminNavigation();
+        CurrencyPageBL currencyPageBL = adminNavigationBL.getToCurrencyPage();
+        currencyPageBL.sortCurrencyByLastUpdateFromOldest();
+        String disabledCurrencyTitle = currencyPageBL.getCurrencyTitle();
+        EditCurrencyPageBL editCurrencyPageBL = currencyPageBL.getToEditCurrencyPage();
+        editCurrencyPageBL.changeStatusOfCurrencyForEnabled();
+
+        new Navigation().navigateToUrl(URLs.BASE_URL.getUrl());
+        MainPageBL mainPageBL = new MainPageBL();
+        HeaderPageBL headerPageBL = mainPageBL.getHeaderPageBL()
+                .clickOnCurrencyDropdownButton()
+                .verifyCurrencyEnabled(disabledCurrencyTitle);
+    }
+}
```

Comment:
Please, put two tests related to currency enabling/disabling into one test class